tutor/src/components/tutor-layout.js

```diff
@@ -117,16 +117,18 @@ class TutorLayout extends React.Component {
     this.courseContext.course = this.props.course;
   }
 
-  /**
-   * Hide the navbar if user is in the 'viewTaskStep' screen.
-   * Use styled-components to check if the screen is mobile width.
-   */
-  shouldHideNavbar() {
+  isViewingTaskStep() {
     const routerName = get(Router.currentMatch(), 'entry.name', '');
     if(routerName === 'viewTaskStep') return true;
     return false;
   }
 
+  isViewingCourseDashboard() {
+    const routerName = get(Router.currentMatch(), 'entry.name', '');
+    if(routerName === 'dashboard') return true;
+    return false;
+  }
```

Comment:
no big thing but I usually prefer to just return the results of the condition since it'll be a boolean:

```return routerName === 'dashboard';```

Reply:
yep, it is more readable. thanks.